### PR TITLE
Code review1 severance

### PR DESCRIPTION
**Pull request by kylerseverance**
The program had limited work done in the master branch to review however looking at the GitHub project it was clear what their app is meant to do and the requirements it will need. 

The program was available in UC Github on time and an invitation was sent in advance, though different from the posted link on Canvas. 

The program was not commented on. The GitHub Projects were very detailed and well documented however there were several "Closed" Projects in the repo that were not found in the code. For example "Adding and Deleting," "Drop Down Menu" and "CheckBox Button" were projects marked as closed but the functions were not found in the code. I reccomend deleting projects that are no longer is use to better organize the workload. 

No errors were found but the project does not complie. 

I added several necessary files that the program will require at some point. I added four buttons that I feel will be needed as well, including a login button or create account button because the app will need to house a lot of user data and an account should be available to house the data. 

https://github.com/JMaz-15/EasySurvey2/blob/CreateSurvey/app/src/main/java/com/example/easysurvey/CreateActivity.kt
https://github.com/JMaz-15/EasySurvey2/blob/a08848cdf75d32b00b26c641d2d494902045514f/app/src/main/res/values/strings.xml
https://github.com/JMaz-15/EasySurvey2/blob/a08848cdf75d32b00b26c641d2d494902045514f/app/src/main/AndroidManifest.xml

### CODE REVIEW

**Comment by kylerseverance** [comment]
During this review I strengthened my understanding on the relationship between the RetrofitClientInstance and the service file. Also by setting up the view model I now better understand how it allows content to reach the MainActivity. I also greatly increased my knowledge on Kotlin coding conventions and feel I no longer will need to second guess my naming conventions and formatting. 

As far as suggestions for the group, I would reccomend creating a dropdown list for the computerComponents similar to how we did for the countries assignment in the main activity. The dao, dto, service, and Retrofit instance are all in place so you should have the foundation for populating a drop down menu once you find the appropriate link for the Retrofit Instance. I would also suggest that you maintain the coding practices displayed thus far. 

I checked very meticulously and found no violations to the kotlinlang.org standtards. All functions fit on a single line, all classes started with upper case and were camel cased, all new lines were seperated appropriately with 4 spaces. Curley brackets all started at the end of a construct, and closed alligning horizontally with the beginning of the construct. I only made one change reguarding naming conventions and that was capitallizing the "R" in RetrofitClientInstance that was in your commented out service file because it eventually needed to be referencing an object file. According to kotlinlang.org, "Names of classes and objects start with an uppercase letter and use camel case."

File: app/src/main/java/com/example/springprojectstationbuild/MainActivity.kt [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        setContent {
+            viewModel.fetchComputerComponents()
+            val components by viewModel.components.observeAsState(initial = emptyList())
+            R.layout.settings_activity
+
+        }
+    }
+}
+@Composable
+fun mainButtons() {
```
Comment (discospiff) [inline app/src/main/java/com/example/springprojectstationbuild/MainActivity.kt:31]: Composable functions diverge from the common practice of lowerCamelCase, and should be nouns, that are PascalCase, believe it or not.

File: app/src/main/java/com/example/springprojectstationbuild/service/ComponentService.kt [hunk 2]
```diff
@@ -1,9 +1,10 @@
-package com.example.springprojectstationbuild.service/*package com.example.springprojectstationbuild.service
```
Comment (discospiff) [inline app/src/main/java/com/example/springprojectstationbuild/service/ComponentService.kt:1]: Good idea to remove commented out code.

File: app/src/main/java/com/example/springprojectstationbuild/service/ComponentService.kt [hunk 3]
```diff
@@ -13,12 +14,12 @@ interface IComponentService {
 class ComponentService : IComponentService {
     override suspend fun fetchComputerComponents(): List<ComputerComponent>? {
         return withContext(Dispatchers.IO){
-            val service = retrofitClientInstance.retrofitInstance?.create(IComponentDAO::class.java)
+            val service = RetrofitClientInstance.retrofitInstance?.create(IComponentDAO::class.java)
```
Comment (discospiff) [inline app/src/main/java/com/example/springprojectstationbuild/service/ComponentService.kt:17]: Good.

**Comment by discospiff** [comment]
Some good suggestions here.  It also appears to introduce new functionality.  The group should ensure this meets their Definition of Done, and then either merge if it does, or keep the branch open, finish the work, and then merge.

**Pull request merged**